diff --git a/smoke_test_data/elixir-1-13/project_with_unformatted_code/lib/project_with_unformatted_code.ex b/smoke_test_data/elixir-1-13/project_with_unformatted_code/lib/project_with_unformatted_code.ex
index 5b65e4d..98f817c 100644
--- a/smoke_test_data/elixir-1-13/project_with_unformatted_code/lib/project_with_unformatted_code.ex
+++ b/smoke_test_data/elixir-1-13/project_with_unformatted_code/lib/project_with_unformatted_code.ex
@@ -2,7 +2,7 @@ defmodule ProjectWithUnformattedCode do
   @moduledoc """
   Documentation for `ProjectWithUnformattedCode`.
 
-  iex> ProjectWithUnformattedCode.add(5,5)
+  iex> ProjectWithUnformattedCode.add(5, 5)
   10
   """
 
@@ -15,7 +15,7 @@ defmodule ProjectWithUnformattedCode do
       3
 
       iex> 1
-      ...>   |> ProjectWithUnformattedCode.add(2)
+      ...> |> ProjectWithUnformattedCode.add(2)
       3
 
   """
@@ -24,12 +24,28 @@ defmodule ProjectWithUnformattedCode do
   end
 
   @doc """
-  iex>   ProjectWithUnformattedCode.subtract( 5, 4 )
+  iex> ProjectWithUnformattedCode.subtract(5, 4)
   1
 
-  iex> [100_000_000_000, 200_000_000_000, 300_000_000_000, 400_000_000_000, 500_000_000_000, 600_000_000_000, 700_000_000_000]
+  iex> [
+  ...>   100_000_000_000,
+  ...>   200_000_000_000,
+  ...>   300_000_000_000,
+  ...>   400_000_000_000,
+  ...>   500_000_000_000,
+  ...>   600_000_000_000,
+  ...>   700_000_000_000
+  ...> ]
   ...> |> Enum.map(&ProjectWithUnformattedCode.subtract(&1, 100_000_000_000))
-  [0, 100_000_000_000, 200_000_000_000, 300_000_000_000, 400_000_000_000, 500_000_000_000, 600_000_000_000]
+  [
+    0,
+    100_000_000_000,
+    200_000_000_000,
+    300_000_000_000,
+    400_000_000_000,
+    500_000_000_000,
+    600_000_000_000
+  ]
   """
   def subtract(a, b) do
     a - b
